src/main/java/dan200/computercraft/core/apis/OSAPI.java

```diff
@@ -153,24 +153,57 @@ private static long getEpochForCalendar( Calendar c )
         return c.getTime().getTime();
     }
 
+    /**
+     * Adds an event to the event queue. This event can later be pulled with
+     * os.pullEvent.
+     *
+     * @param name The name of the event to queue.
+     * @param args The parameters of the event.
+     * @cc.param [opt] ... The parameters of the event.
```

Comment:
I've generally taken the approach that variadic arguments don't need to be `[opt]` - you can assume you can get away with passing 0 arguments to them.

I have a feeling you'll also need a `@cc.param string name` here. It's kind of an "all or nothing" approach.